lib/facter/resolvers/networking.rb

```diff
@@ -71,7 +71,8 @@ def extract_mac(raw_data, parsed_interface_data)
         end
 
         def extract_dhcp(interface_name, raw_data, parsed_interface_data)
-          return unless raw_data =~ /status:\s+active/
+          return unless ((raw_data =~ /status:\s+active/) &&
+                         (OsDetector.instance.identifier == :windows))
```

Comment:
`ipconfig` is valid on macOS too:

```
# uname -a
Darwin taxable-stump 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64 x86_64
# ipconfig getoption en0 server_identifier
10.32.22.9
```

Reply:
Candidly, I'm not familiar with the OS X environment, or so much of Windows networking. I was aware that there is an ipfconfig tool on Windows, did not take a look at the cmd syntax though.

It looks like :macosx is actually where the matching ipconfig would be found. I'll update the patch